Library/Homebrew/cmd/info.rb

```diff
@@ -68,6 +69,20 @@ def info
   end
 
   def print_info
+    if args.days.present?
+      valid_days = %w[30 90 365]
+      raise UsageError, "days must be one of #{valid_days.join(", ")}" unless valid_days.include?(args.days)
+    end
+    if args.category.present?
+      valid_formula_categories = %w[install install-on-request build-error]
+      valid_categories = valid_formula_categories + %w[cask-install os-version]
+      if ARGV.named.present? && !valid_formula_categories.include?(args.category)
```

Comment:
I believe `args.remaining` is equivalent to `ARGV.named`, could be good to use it here since `ARGV` is being deprecated.

Reply:
I'd like to humbly disagree for now until we've moved across to it properly.